app/jobs/update_jobs/adjust_question_reflections.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class UpdateJobs::AdjustQuestionReflections < CloneJob
+  def perform(question, prev_variable_name)
+    reflectable_questions(question).each do |q|
+      q.narrator['blocks'].each do |block|
+        next if block['reflections'].nil?
+
+        block['reflections'].each do |reflection|
+          next unless reflection['variable'] == prev_variable_name
+
+          reflection['variable'] = question.body['variable']['name']
+        end
+
+        q.save!
```

Comment:
Here you can also use Postgresql operators/functions on jsonb to speed up a little bit this operation. It is a potential bottleneck